Common/src/Pollster/TaskHandler.cs

```diff
@@ -580,42 +581,50 @@ await agent_.FinalizeTaskCreation(CancellationToken.None)
     }
     catch (Exception e)
     {
-      await HandleErrorAsync(e,
-                             taskData_,
-                             cancellationTokenSource_.Token)
+      await HandleErrorResubmitAsync(e,
+                                     taskData_,
+                                     cancellationTokenSource_.Token)
         .ConfigureAwait(false);
     }
 
     logger_.LogDebug("End Task Processing");
   }
 
-  private async Task HandleErrorAsync(Exception         e,
-                                      TaskData          taskData,
-                                      CancellationToken cancellationToken)
+  private async Task HandleErrorRequeueAsync(Exception         e,
```

Comment:
I wonder if it would be better to make this method just return the task from internal, instead of making this function asynchronous and awaiting it.
This does not block the PR anyway.

Reply:
I don't want to try the propogation of exceptions in async tasks.